app/src/test/java/org/oppia/android/app/topic/questionplayer/QuestionPlayerActivityLocalTest.kt

```diff
@@ -216,6 +263,27 @@ class QuestionPlayerActivityLocalTest {
     return scrollToHolder(StateViewHolderTypeMatcher(viewType))
   }
 
+  // Returns a matcher that matches for active confetti.
+  private fun hasActiveConfetti(): ViewAssertion {
+    return StateFragmentActiveConfettiAssertion()
+  }
+
+  // Custom class to check if a KonfettiView isActive().
+  private class StateFragmentActiveConfettiAssertion() : ViewAssertion {
+    override fun check(view: View, noViewFoundException: NoMatchingViewException?) {
+      if (noViewFoundException != null) {
+        throw noViewFoundException
+      }
```

Comment:
Could we maybe use a ``TypeSafeMatcher`` instead? I think that will automatically check against the ``KonfettiView`` type.

Reply:
Done